packages/react/src/primitives/types/toggleButtonGroup.ts

```diff
@@ -34,5 +34,5 @@ export interface ToggleButtonGroupProps
    * @description
    * Handles changes to the current value when using the ToggleButtonGroup as a controlled component
    */
-  onChange: (value: string | string[]) => void;
+  onChange: (value: string | Array<string | undefined> | undefined) => void;
```

Comment:
Nit: prefer shorthand `Array` syntax
```suggestion
  onChange?: (value: string | (string | undefined)[] | undefined) => void;
```